docs/csharp/tutorials/microservices.md

```diff
@@ -354,125 +358,130 @@ the machine parameters when you start, you only need to
 supply any changes. The changes here will be to include
 our application.
 
-In this first sample, we'll use the `onbuild` version of
-the RC2 image. This is the easiest way to create a working Docker
-environment. However, the image it creates is larger than necessary.
-This image include the dotnet core runtime, and the dotnet SDK. 
+In this first sample, we'll use the `latest` version of
+the dotnet image. This is the easiest way to create a working Docker
+environment. This image include the dotnet core runtime, and the dotnet SDK. 
+That makes it easier to get started and build, but does create a larger image.
 
-The next two lines load SQLite onto the machine:
-
-```
-RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sources.list
-RUN apt-get -qq update && apt-get install -qqy sqlite3 libsqlite3-dev && rm -rf /var/lib/apt/lists/*
-```
-
-We're not using SQLite, but leave it in place for reference if you need it later.
-
-The next three lines setup your application:
+The next four lines setup and build your application:
 
 ```
 COPY . /app
 WORKDIR /app
 RUN ["dotnet", "restore"]
+RUN ["dotnet", "build"]
 ```
 
 This will copy the contents of the current directory to the docker VM, and restore
-all the packages.
+all the packages. Using the dotnet CLI means that the Docker image must include the
+.NET Core SDK. 
 
-The final lines of the file set the output port (80) and run the application:
+The final lines of the file set the tcp port (5000) this container 
+listens on and runs the application:
 
 ```
-EXPOSE 80
-ENTRYPOINT ["dotnet", "run"]
+EXPOSE 5000/tcp
+ENTRYPOINT ["dotnet", "run", "--server.urls", "http://0.0.0.0:5000"]
 ```
 
-Notice that this Dockerfile uses the dotnet cli to build and run your docker image.
-That's why the larger image is needed.
+The Dockerfile uses declares port 5000 here because it's the default port that an
```

Comment:
Maybe something like: Using the EXPOSE command informs Docker to listen on port 5000. This is the default port for ASP.NET Core applications and must be defined in the ENTRYPOINT command using the `--server.urls` command line option.  Alternatively you may also define this in the `WebHostBuilder.UseUrls('http://0.0.0.0:5000')` method.